composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "type": "project",
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.0",
```

Comment:
Let's please use `^8.0` - we don't know if PHP 9.0 will be compatible :D 

I also endorse `~8.0 || ~8.1`, as explained in https://github.com/laminas/laminas-diactoros/issues/117#issuecomment-1267306142

Reply:
@guidobonuzzi important to fix this previous to merge BTW